mastering-plone/events.rst

```diff
@@ -12,7 +12,7 @@ Turning Talks into Events
         git checkout events
 
 
-We forgot something: A list of talks is great especially if you can sort it by your preferences. But if a visitor decides he wants to actually go to see a talk he needs to know when it will take place.
+We forgot something: a list of talks is great, especially if you can sort it according to your preferences. But if a visitor decides she wants to actually go to see a talk she needs to know when it will take place.
```

Comment:
Personally I prefer the neutral pronouns of "they/them/theirs". This may be something to discuss at an organizational level?

Reply:
OK but I have seen quite a few she/hers in this and the main docs.